packages/server/customer-os-neo4j-repository/repository/domain_write_repository.go

```diff
@@ -0,0 +1,62 @@
+package repository
+
+import (
+	"context"
+	"fmt"
+	"github.com/neo4j/neo4j-go-driver/v5/neo4j"
+	"github.com/openline-ai/openline-customer-os/packages/server/customer-os-common-module/utils"
+	"github.com/openline-ai/openline-customer-os/packages/server/customer-os-neo4j-repository/tracing"
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/log"
+	"time"
+)
+
+type DomainWriteRepository interface {
+	CreateDomain(ctx context.Context, domain, source, appSource string, now time.Time) error
+}
+
+type domainWriteRepository struct {
+	driver   *neo4j.DriverWithContext
+	database string
+}
+
+func (d domainWriteRepository) CreateDomain(ctx context.Context, domain, source, appSource string, now time.Time) error {
```

Comment:
if time is sent as parameter rename it from now, as it can be any value. to avoid confusion